packages/authorization-process/src/services/validators.ts

```diff
@@ -1,27 +1,33 @@
 import { userRoles } from "pagopa-interop-commons";
-import { TenantId, UserId } from "pagopa-interop-models";
-import { selfcareV2Client } from "pagopa-interop-selfcare-v2-client";
+import { TenantId, UserId, Client } from "pagopa-interop-models";
+import { SelfcareV2Client } from "pagopa-interop-selfcare-v2-client";
 import { userWithoutSecurityPrivileges } from "../model/domain/errors.js";
-
-export const isClientConsumer = (
-  consumerId: TenantId,
-  organizationId: string
-): boolean => (consumerId === organizationId ? true : false);
+import { organizationNotAllowedOnClient } from "../model/domain/errors.js";
 
 export const assertUserSelfcareSecurityPrivileges = async (
   selfcareId: string,
   requesterUserId: UserId,
-  userId: UserId
+  consumerId: TenantId,
+  selfcareV2Client: SelfcareV2Client
 ): Promise<void> => {
   const users = await selfcareV2Client.getInstitutionProductUsersUsingGET({
     params: { institutionId: selfcareId },
     queries: {
       userIdForAuth: requesterUserId,
-      userId,
+      userId: consumerId,
```

Comment:
the `consumerId` is a tenant, here we need the `userId`
```suggestion
      userId: userId,
```